Makefile

```diff
@@ -127,6 +128,10 @@ endif
 
 ifeq ($(OS),macosx)
 LD_FLAGS += -undefined dynamic_lookup
+LD_FLAGS += \
+	-framework CoreFoundation \
+	$(GETTEXT_PREFIX)/lib/libintl.a \
```

Comment:
Note that GETTEXT_PREFIX is set within `paella/mk/macosx.def` to be either `/usr/local` (for our custom build) or to the result of `brew --prefix gettext` (if we get the package with `brew`).

Reply:
Its only macos related right?

Reply:
Yes, we don't currently install `libintl.a` (via `gettext`) for Linux. The reason we install it for macOS is that having the wrong `gettext` around (like the one you may get with `brew`) may break the build and there's no simple way of telling `cpython` to ignore `gettext` altogether.